openstudiocore/ruby/openstudio_cli.rb

```diff
@@ -1,3 +1,13 @@
-load "#{File.dirname(__FILE__)}/openstudio.rb"
+require "openstudio-workflow"
 
-puts 'hello world'
+osw_path = ARGV[0]
```

Comment:
I ran from `E:\openstudio-2-0\build\OSCore-prefix\src\OSCore-build\ruby\Debug` the command:

    \openstudio-2-0\build\Ruby-prefix\src\Ruby\bin\ruby.exe -I . openstudio_cli.rb \openstudio-workflow-gem\spec\files\compact_osw\compact.osw

I got this error:

    E:/openstudio-2-0/build/Ruby-prefix/src/Ruby/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- parallel (LoadError)
        from E:/openstudio-2-0/build/Ruby-prefix/src/Ruby/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require'
        from E:/openstudio-workflow-gem/spec/files/compact_osw/measures/DencityReports/measure.rb:545:in `run'
        from E:/openstudio-2-0/build/Ruby-prefix/src/Ruby/lib/ruby/gems/2.0.0/gems/openstudio-workflow-0.2.0/lib/openstudio/workflow/util/measure.rb:312:in `apply_measure'